src/components/Modals/FiatRamps/fiatRampProviders/banxa.ts

```diff
@@ -0,0 +1,38 @@
+import { adapters } from '@shapeshiftoss/caip'
+import queryString from 'querystring'
+
+import { FiatRampAction, FiatRampAsset } from '../FiatRampsCommon'
+
+export const getBanxaAssets = () => {
+  const banxaAssets = adapters.getSupportedBanxaAssets()
+  const assets: FiatRampAsset[] = banxaAssets.map(asset => ({
+    assetId: asset.CAIP19,
```

Comment:
```suggestion
    assetId: asset.assetId,
```

Reply:
blocked by https://github.com/shapeshift/lib/pull/624

Reply:
@0xdef1cafe please approve the lib PR.